simulator.py

```diff
@@ -435,6 +457,9 @@ def _select_location(self, location_type, city):
             locs = city.stores
             visited_locs = self.visits.stores
 
+        elif location_type == "hospital":
+            raise NotImplementedError("hospital location selection not implemented yet")
```

Comment:
It can be simply implemented as closest vacant hospital relative to current location

Reply:
fixed. I think the vacancy of hospital is automatically checked by Simpy because hospital is a location and all locations are Simpy.Resource so if the capacity reaches they are put in waitlist.